examples/README.md

```diff
@@ -0,0 +1,11 @@
+Examples of building an image to run on LinuxKit or on a host
+
+```
+moby build -output docker -o - docker.yml nginx.yml | docker build -t dockertest -
+docker run -d -p 80:80 --privileged dockertest
+
+moby build -output kernel+initrd linuxkit.yml nginx.yml
+linuxkit run nginx
+```
+
+Both of these will run the same `nginx` either in a VM or a container.
```

Comment:
I think it would be useful to add a bit more info on how this works. By the looks of it you seem to start a alpine container in docker in which containerd is started and then init/service containers are run.